v03_pipeline/lib/paths.py

```diff
@@ -328,7 +331,7 @@ def valid_reference_dataset_collection_path(
 
 def valid_reference_dataset_path(
     reference_genome: ReferenceGenome,
-    reference_dataset: ReferenceDataset,
+    reference_dataset: 'ReferenceDataset',
```

Comment:
There's an annoying circular dependency that gets triggered because of the type import.  Restructuring felt out of scope for this.

Reply:
when I first created the `ReferenceDataset` enum it was challenging to figure out how to avoid this